app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -34,6 +34,21 @@ $large-input-size: 50px;
   }
 }
 
+%gem-c-search-input-focus {
```

Comment:
Funnily, it seems like this is what I had originally done in the PR that introduced this warning, but it was suggested to extend the original selector instead 😆 
 https://github.com/alphagov/govuk_publishing_components/pull/1752#discussion_r512567735

Reply:
@danacotoran ah ha that's good to see. It looks like it was very hard to anticipate that this would have the additional effects it did (I thought I was only removing a warning when I looked at this, I had no idea it produced 59 extra code outputs)